server/src/score/workshop-score.entity.ts

```diff
@@ -0,0 +1,29 @@
+import { Application } from '../application/application.entity';
+import { ScoreBaseEntity } from '../common/score-base.entity';
+import { User } from '../user/user.entity';
+import { Entity, PrimaryGeneratedColumn, OneToOne, JoinColumn, ManyToOne, Column } from 'typeorm';
+import { WorkshopScoreStatus } from '../common/enums';
+
+@Entity({
+  name: 'pbgp_workshop_score',
```

Comment:
We should go through their Database documentation and update these. I will schedule a meeting with Iouri regarding this!

Reply:
This one will have to wait it seems, haha!